engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -233,8 +230,11 @@ class MetadataBuilderActor(serviceRegistryActor: ActorRef) extends LoggingFSM[Me
     case Event(failure: WorkflowQueryFailure, _) =>
       context.parent ! RequestComplete(StatusCodes.BadRequest, APIResponse.fail(failure.reason))
       allDone
-    case Event(WorkflowOutputsResponse(w, metadata), _) =>
-      context.parent ! RequestComplete(StatusCodes.OK, workflowMetadataResponse(w, metadata, includeCallsIfEmpty = false))
+    case Event(WorkflowOutputsResponse(id, events), _) =>
+      // Add in an empty output event if there aren't already any output events.
+      val hasOutputs = events collectFirst { case MetadataEvent(MetadataKey(_, _, key), _, _) if key == WorkflowMetadataKeys.Outputs => () } isDefined
+      val updatedEvents = if (hasOutputs) events else MetadataEvent.empty(MetadataKey(id, None, WorkflowMetadataKeys.Outputs)) +: events
```

Comment:
Ok, if we start having a lot of those "mandatory even if empty" fields in the response we might want to have some kind of template with all of them (either Json or as MetadataEvents), and add the actual events on top of that. But that's not for this PR